### PR TITLE
fix ai actions

### PR DESCRIPTION
**Pull request by VietnamNeko**

### CODE REVIEW

File: src/gameClasses/components/script/ActionComponent.js [hunk 1]
```diff
@@ -1275,7 +1275,7 @@ var ActionComponent = TaroEntity.extend({
 						var value = self._script.variable.getValue(action.number, vars);
 						if (unit) {
 							if (!isNaN(value)) {
-								unit.ai.maxAttackRange = value;
+								unit._stats.ai.maxAttackRange = value;
```
Comment (sloont) [inline src/gameClasses/components/script/ActionComponent.js:1278]: is there already an `ai` object under `_stats`?

Comment (sloont) [reply src/gameClasses/components/script/ActionComponent.js:1278]: also:  will these changes break anything existing?

Comment (VietnamNeko) [reply src/gameClasses/components/script/ActionComponent.js:1278]: yeah there is. check AIcomponent

Comment (VietnamNeko) [reply src/gameClasses/components/script/ActionComponent.js:1278]: no, they won't break anything, rn the actions aren't working

**Review by sloont — APPROVED** [review]

**Pull request merged**